vault-upgrade-hook/pom.xml

```diff
@@ -109,6 +109,11 @@
             <artifactId>slf4j-api</artifactId>
             <version>1.7.7</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.2</version>
```

Comment:
Hi, two thoughts about this dependency:
- is it compatible with AEM 6.0?
- and we should make it optional since it is only needed for SlingPipes and UserPreference actions right?

Reply:
Use some kind of sorted map and serialize it to String with toString().